server/database.py

```diff
@@ -22,13 +22,14 @@ def get_id(self):
 
 
 class User(AsceeModel, UserMixin):
-    is_admin = props.Bool(default=False)
-    is_recruiter = props.Bool(default=False)
-    is_senior_recruiter = props.Bool(default=False)
+    is_admin = props.Bool(indexed=True, default=False)
+    is_recruiter = props.Bool(indexed=True, default=False)
+    is_senior_recruiter = props.Bool(indexed=True, default=False)
     recruiter_id = props.Integer(indexed=True, optional=True)
     status_level = props.Integer(indexed=True, default=0)
+    name = props.String(default="Unknown")
 
-    STATUS_LIST = ('new', 'escalated', 'accepted', 'rejected')
+    STATUS_LIST = ('new', 'claimed', 'escalated', 'accepted', 'rejected')
```

Comment:
As I pointed out in the other PR, 'claimed' shouldn't be a status since this info is already present in `recruiter_id`. Bad DB form to duplicate information!

Reply:
it should on the API, whether it is in DB or not

Reply:
OK, I can modify the get_applicants code to use 'claimed' as a status if a user is new and has a recruiter_id.